src/librustc_typeck/collect.rs

```diff
@@ -1266,48 +1267,40 @@ fn impl_polarity<'a, 'tcx>(tcx: TyCtxt<'a, 'tcx, 'tcx>,
     }
 }
 
-// Is it marked with ?Sized
-fn is_unsized<'gcx: 'tcx, 'tcx>(astconv: &AstConv<'gcx, 'tcx>,
+// Is it marked with ?Move/?Sized
+fn opt_out_bounds<'gcx: 'tcx, 'tcx>(astconv: &AstConv<'gcx, 'tcx>,
```

Comment:
nit: move the bounds into a struct rather than passing a `(bool, bool)`?

Reply:
I created a struct for this and also made `Bounds` and `compute_bounds` use it.